superset/assets/javascripts/modules/utils.js

```diff
@@ -203,3 +203,11 @@ export function initJQueryAjaxCSRF() {
     });
   }
 }
+
+export function getTextWidth(text, fontDetails = '12px Roboto') {
```

Comment:
to pull the table out of superset, we'd need to pull this out as well. maybe there could be a table utils file?
also any idea how expensive this is? could the canvas be cached?